rotkehlchen/chain/optimism/modules/velodrome/decoder.py

```diff
@@ -36,11 +64,224 @@ def __init__(
         )
         self.pools, self.gauges = read_velodrome_pools_and_gauges_from_cache()
 
-    def _decode_velodrome_events(self) -> None:
-        pass
+    def _decode_add_liquidity_events(
+            self,
+            tx_log: 'EvmTxReceiptLog',
+            decoded_events: list['EvmEvent'],
+    ) -> DecodingOutput:
+        """Decodes events that add liquidity to a velodrome pool"""
+        for event in decoded_events:
+            try:
+                crypto_asset = event.asset.resolve_to_crypto_asset()
+            except (UnknownAsset, WrongAssetType):
+                self.notify_user(event=event, counterparty=CPT_VELODROME)
+                raise
```

Comment:
As you can check in https://github.com/rotki/rotki/blob/1169de3db17ef203a34d87a5fa49b8e54264ffe9/rotkehlchen/chain/evm/decoding/decoder.py#L362 we don't handle the wrongAssetType error. Should be there too